x-pack/plugins/cloud_security_posture/server/create_indices/create_indices.ts

```diff
@@ -84,20 +86,25 @@ const createLatestFindingsIndex = async (esClient: ElasticsearchClient, logger:
     const { template, composed_of, _meta } =
       findingsIndexTemplateResponse.index_templates[0].index_template;
 
-    if (template?.settings) {
-      template.settings.lifecycle = {
-        name: '',
-      };
-    }
-
     // We always want to keep the index template updated
     await esClient.indices.putIndexTemplate({
       name: LATEST_FINDINGS_INDEX_TEMPLATE_NAME,
       index_patterns: LATEST_FINDINGS_INDEX_PATTERN,
       priority: 500,
+      template: {
+        mappings: template?.mappings,
+        settings: {
+          ...template?.settings,
+          default_pipeline: CSP_LATEST_FINDINGS_INGEST_TIMESTAMP_PIPELINE,
+          lifecycle: {
+            ...template?.settings?.lifecycle,
+            name: '',
+          },
+        },
+        aliases: template?.aliases,
```

Comment:
we should not have the same aliases between findings index and findings latest index

Reply:
@CohenIdo pointed me that I'm a bit wrong here, it is as it was before regarding the `aliases`.

However, we should note that now we are in opt-in state of mind instead of opt-out, any fields that are part of the template. Worth noting that.

Reply:
So should I keep it?